### ISSUE TITLE
Default colors?

### ISSUE THREAD

**Issue by max-sixty** [zellij-org/zellij#886]
I'm trying to change one color but zellij complains that `fg` is not listed. What do you think about either allowing the others to default, or making it easy for a user to grab the default colors? This could be as part of the [docs](https://zellij.dev/documentation/themes.html) or `zellij setup --dump-config`.

Also — if anyone has a decent scheme for light colors I'd be keen to copy it! I don't have strong preferences so happy to take anything reasonable. Currently the colors are very difficult to read in light mode.

**Comment by imsnif** [comment]
I agree with all your suggestions! I'd also be happy to merge a light theme into main.

**Comment by casonadams** [comment]
why not just use the ANSI colors of the terminal?  That way a color theme can be defined as `active-pane-border: green` and it will always look good because the terminal colors will just match.

**Comment by casonadams** [comment]
Actually you can do something like this with the current version:

<img width="766" alt="image" src="https://user-images.githubusercontent.com/17597548/155455747-5004f91f-21f2-4df9-aac8-b63ea5484cb0.png">

<img width="766" alt="image" src="https://user-images.githubusercontent.com/17597548/155455789-0745cac5-2733-4f61-88d2-5ec2a9be7eb6.png">

<img width="766" alt="image" src="https://user-images.githubusercontent.com/17597548/155455800-e942b395-1924-4f08-805d-586135c3b4c1.png">

Here is what I am doing.

If using alacritty use [alacritty-shell](https://github.com/casonadams/alacritty-shell)
Or for a more generic utility [wall-shell](https://github.com/casonadams/walh-shell)

Then setup the theme:

```yaml
themes:
  default:
    fg: 7
    bg: 0
    black: 0
    red: 1
    green: 2
    yellow: 3
    blue: 4
    magenta: 5
    cyan: 6
    white: 7
    orange: 208
    gray: 247

theme: default
```

Note if you don't want to use the utilities above just make sure that the gray is a color that looks good with the theme you are using.

**Comment by jaeheonji** [comment]
Since the issue is outdated and the theme system has been migrated, I'm closing this issue.
